app/javascript/products.vue

```diff
@@ -324,14 +338,45 @@ export default {
         })
       return params
     },
-    countProductsGroupedBy({ elapsed_days: elapsedDays }) {
+    getElementNdaysPassed(elapsedDays) {
       const element = this.productsGroupedByElapsedDays.find(
         (el) => el.elapsed_days === elapsedDays
       )
+      return element
+    },
+    countProductsGroupedBy({ elapsed_days: elapsedDays }) {
+      const element = this.getElementNdaysPassed(elapsedDays)
       return element === undefined ? 0 : element.products.length
     },
     elapsedDaysId(elapsedDays) {
       return `${elapsedDays}days-elapsed`
+    },
+    PassedAlmost5daysProducts(products) {
+      const productsPassedAlmost5days = products.filter((product) => {
+        const thresholdDay = 5
+        const thresholdHour = 8
+        return (
+          Math.floor((thresholdDay - this.elapsedTimes(product)) * 24) <=
+          thresholdHour
+        )
+      })
+      return productsPassedAlmost5days
+    },
+    elapsedTimes(product) {
+      const lastSubmittedTime =
+        product.published_at_date_time || product.created_at_date_time
+      const elapsedTimes =
+        (new Date() - new Date(lastSubmittedTime)) / 1000 / 60 / 60 / 24
+      return elapsedTimes
```

Comment:
```suggestion
      const lastSubmittedTime =
        product.published_at_date_time || product.created_at_date_time
      return (new Date() - new Date(lastSubmittedTime)) / 1000 / 60 / 60 / 24
```

elapsedTimesという変数をいちいち作ってしまうとメソッドにした意味が薄れるのでこういう感じがいいと思います。